src/chat.rs

```diff
@@ -1440,15 +1483,6 @@ impl Chat {
         let mut to_id = 0;
         let mut location_id = 0;
 
-        if let Some(reason) = self.why_cant_send(context).await? {
-            if self.typ == Chattype::Group && reason == CantSendReason::NotAMember {
-                context.emit_event(EventType::ErrorSelfNotInGroup(
-                    "Cannot send message; self not in group.".into(),
-                ));
-            }
-            bail!("Cannot send message to {}: {}", self.id, reason);
-        }
-
         let from = context.get_primary_self_addr().await?;
```

Comment:
FTR, I removed this in commit bdf9a338ec47d99bf3ba9d578d6b784234d3202c

    Bugfix
    
    The bug was:
    
    - Scan your chat partner's QR Code
    - They change devices
    - They send you a message
    - Without accepting the encryption downgrade, scan your chat partner's QR Code again
    
    -> The re-verification fails.

This code was not helpful because all the sites that call `prepare_msg_raw()` check `can_send`/`why_cant_send` themselves, so I just removed it.